src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageCredentialsValidator.cs

```diff
@@ -88,5 +96,23 @@ internal class DefaultStorageCredentialsValidator : IStorageCredentialsValidator
                 throw;
             }
         }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "IsDevStoreAccount")]
```

Comment:
nit: can you move this to the GlobalSuppressions? That's where we keep all our supressions.

Reply:
I see this in GlobalSuppressions now, but did you forget to remove it from here?